src/SuperMint.sol

```diff
@@ -7,11 +7,15 @@ import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
 contract SuperMint is IERC721Receiver {
   MaxMint721 implementation;
   uint maxMints;
-  bool complete;
+  //added a new variable to perform the iteration and removed the bool variable
```

Comment:
i'm just being picky but can u pls add a space after '//', e.g. '// added...' for every comment